frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -73,6 +73,18 @@ export class TemplateValidations {
     });
   };
 
+  // Return a valid bus, return the defaultBus if it's valid.
+  getAValidBus = (defaultBus?: DiskBus): DiskBus => {
```

Comment:
can we rename it to `getDefaultBus` and have zero arguments here?

Reply:
I'd like to keep this method (we can change the name if you like)
the purpose is to verify the bus we are using as default, thus is has to have an argument.
I except the comment to have an additional method, with no arguments, to get the first valid bus